### DISCUSSION TITLE
What does prefix ooo doing

### DISCUSSION THREAD

**Discussion by Tusenka** [prometheus/prometheus#15088] [Q&A]
HELP!!!
https://github.com/prometheus/prometheus/blob/199d2fb492322af138ef85d08ba022066fea784c/tsdb/db.go#L1135

**Comment by bwplotka** [comment]
Hi, those types of questions are bit small for a GH discussion. Plus this code might change tomorrow and the answer depends on what you are trying to do. Are you on CNCF Slack @Tusenka ? Happy to help offline and discuss next steps 🤗 I'm `@bwplotka` there, or you can ping me on `#prometheus-dev` (see https://prometheus.io/community/#:~:text=Slack%20channel%3A%20%23prometheus%2Ddev)
